src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/CaseAllocation.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data
+
+data class CaseAllocation(
```

Comment:
this is only used in tests, should probably be in src/test/

Reply:
This is also used in POST as well, so not moving to test